tests/steinberger_plasticity_1.prm

```diff
@@ -0,0 +1,105 @@
+# This is a test for using Drucker-Prager plasticity with the Steinberger material model.
+# Model is set up in a more realistic Earth-like condition.
+
+set Dimension                              = 2
+set Use years in output instead of seconds = true
+set End time                               = 1e4
+set Adiabatic surface temperature         = 1600.0
+set Output directory                       = output-steinberger_plasticity_1
+set Nonlinear solver scheme                = iterated Advection and Stokes
+
+subsection Formulation
+  set Mass conservation = projected density field
+  set Temperature equation = real density
+end
+
+# Composition setup
+subsection Compositional fields
+  set Number of fields = 1
+  set Names of fields  = density_field
+  set Types of fields  = density
+  set Compositional field methods = prescribed field
+end
+
+subsection Material model
+  set Model name = Steinberger 
+
+  subsection Steinberger model
+    set Data directory                   = $ASPECT_SOURCE_DIR/data/material-model/steinberger/test-steinberger-compressible/
+    set Material file names              = testdata.txt
+
+    # No temperature dependency of viscosity
+    set Lateral viscosity file name      = test-viscosity-prefactor.txt
+
+    # eta_ref = 1e24 Pa s at all depth
+    set Radial viscosity file name       = test-radial-visc-higher.txt
+
+    set Bilinear interpolation           = true
+    set Latent heat                      = false
+    set Thermal conductivity             = 1.5
+    set Maximum viscosity                = 1e24
+
+    # Drucker-Prager plasticity
+    set Cohesion                         = 1e7
+    set Angle of internal friction       = 0.01 # = 0.01 radians
```

Comment:
This will change to 
```suggestion
    # Drucker-Prager plasticity
    set Cohesions                         = 1e7
    set Angles of internal friction       = 0.01 # = 0.01 radians
```
when you use the Drucker Prager parameters.